filter.php

```diff
@@ -52,6 +52,34 @@ public function filter($text, array $options = []) {
         return $this->get_translation($text, $language, $format);
     }
 
+    private function containsmlangtags($text) {
```

Comment:
1. This should better be named something like `contains_multilang_tags()`
2. It should not be private. Private methods are difficult / impossible to be unit-tested in testable subclasses. I would make it protected or public method and have some unit tests for it.

Reply:
Fixed. I didn't spend time writing unit tests initially as I took the code from core, but would definitely be nice to have.

Reply:
> I took the code from core

Can you recall where it was used?

Reply:
Sorry, not core, the mlang v2 filter.